src/main/steps/common/summary-international/utils.ts

```diff
@@ -137,3 +118,38 @@ export const getSelectedPrivateDetails = (userCase: Partial<CaseWithId>): string
   tempDetails = tempDetails + '</ul>';
   return tempDetails;
 };
+
+function getFieldValue(fieldTypes: any, key: string, userCase: Partial<CaseWithId>, language: string | undefined) {
```

Comment:
Where is the welsh translation for the values?

Reply:
As the key is getting passed here .. it will be in english .. but value will be based on the language we select.